addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -1623,6 +1623,10 @@ export class OdooEditor extends EventTarget {
             }
         }
         if (editorCommands[method]) {
+            // Make sure to restore the content editable before applying an
+            // editor command, as it might have been temporarily disabled for
+            // browser behaviors which should not concern editor commands.
+            this._activateContenteditable();
```

Comment:
I'm confused. We re-enable it but we don't re-disable it at some point ?

Reply:
In the use case we had it got reactivated after the command was called but I did just find some cases where it doesn't so I'm gonna fix that.